### PR TITLE
Make execute more useful and not crash on success

### PR DESCRIPTION
**Pull request by jyapayne**

### CODE REVIEW

File: src/webdriver.nim [hunk 1]
```diff
@@ -250,22 +258,20 @@ proc internalExecute(self: Session, code: string, args: varargs[JsonNode], kind:
...
     raise newException(JavascriptException, respObj["value"]["message"].getStr & "\n" & respObj["value"]["stacktrace"].getStr)
 
   return respObj["value"]
 
-proc execute*(self: Session, code: string, args: varargs[JsonNode]): JsonNode =
+proc execute*(self: Session, code: string, args: varargs[JsonNode, `%`]): JsonNode =
   self.internalExecute(code, args, "sync")
 
-proc executeAsync*(self: Session, code: string, args: varargs[JsonNode]): JsonNode =
+proc executeAsync*(self: Session, code: string, args: varargs[JsonNode, `%`]): JsonNode =
```
Comment (dom96) [inline src/webdriver.nim]: Implicitly converting any argument to a JSON node seems risky here. I'd prefer if the user is explicit.

**Review by dom96 — CHANGES_REQUESTED** [review]
Mostly looks good, just one thing :)

Thanks for the PR!

File: src/webdriver.nim (hunk 1 above)
Comment (jyapayne) [reply src/webdriver.nim]: Sure. I'll make two explicit procs for the `Element` object

File: src/webdriver.nim [hunk 2]
```diff
@@ -268,6 +274,18 @@ proc execute*(self: Session, code: string, args: varargs[JsonNode]): JsonNode =
 proc executeAsync*(self: Session, code: string, args: varargs[JsonNode]): JsonNode =
   self.internalExecute(code, args, "async")
 
+proc execute*(self: Session, code: string, args: varargs[Element]): JsonNode =
+  var newArgs = newSeq[JsonNode](args.len)
```
Comment (jyapayne) [inline src/webdriver.nim]: Is there a better way to copy over the args?

Comment (dom96) [reply src/webdriver.nim]: ``@args`` might work, although you need to map over it anyway to apply `%` so you can use `map` I guess

Comment (jyapayne) [reply src/webdriver.nim]: Cool, done

**Review by dom96 — APPROVED** [review]
Nice!

**Pull request merged**